scripts/start/prepareAndRun.R

```diff
@@ -19,17 +19,16 @@ prepareAndRun <- function() {
     # If no "full.gms" exists, the script assumes that REMIND did not run before and
     # prepares all inputs before starting the run.
     prepare()
-    start_subsequent_runs <- TRUE
   } else {
     # If "full.gms" exists, the script assumes that a full.gms has been generated before and you want
     # to restart REMIND in the same folder using the gdx that it eventually previously produced.
     message("\nRestarting REMIND, find old log in 'log_beforeRestart.txt'.")
-    if(file.exists("fulldata.gdx")) file.copy("fulldata.gdx", "input.gdx", overwrite = TRUE)
-    start_subsequent_runs <- FALSE
+    if (file.exists("log.txt")) file.copy("log.txt", "log_beforeRestart.txt", overwrite = TRUE)
+    if (file.exists("fulldata.gdx")) file.copy("fulldata.gdx", "input.gdx", overwrite = TRUE)
```

Comment:
I agree with what you write in the conversation tab: this copy command comes too late, because the `log.txt` has already been recreated in [`submit.R`](https://github.com/remindmodel/remind/blob/2758f32fe4907ad33f575f45aff497580f949012/scripts/start/submit.R#L133). Do you think it would be correct to move the copy command [here](https://github.com/remindmodel/remind/blob/2758f32fe4907ad33f575f45aff497580f949012/scripts/start/submit.R#L36)?